inventory-service/src/main/java/com/petclinic/inventoryservice/datalayer/Bundle.java

```diff
@@ -0,0 +1,23 @@
+package com.petclinic.inventoryservice.datalayer;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import lombok.*;
+import org.springframework.data.annotation.Id;
+
+import java.util.Date;
+import java.util.UUID;
+
+@Data
+@ToString
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class Bundle {
+
+    @Id
```

Comment:
the id annotation defined here is not properly used. You should assign this to a parameter id which will be the privately stored id and the bundleUUID should be the one being accessed and not be annotated with @Id